check-mysql/lib/readonly.go

```diff
@@ -0,0 +1,47 @@
+package checkmysql
+
+import (
+	"os"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/mackerelio/checkers"
+)
+
+type readOnlyOpts struct {
+	mysqlSetting
+	NoReadOnly bool `short:"n" long:"no-readonly" description:"expect that it is not read_only with the MySQL parameter"`
+}
+
+func isReadOnly(v string) bool {
```

Comment:
This function looks weird to me... maybe inlining is better for this...

Reply:
ON / OFF of character string is dependent on MySQL side. On the program side I had the reason that I wanted to convert it once and treat it as True / False.

There is no strong reason so I will fix it. (inlining)